TUTORIAL.md

```diff
@@ -78,4 +78,140 @@ D. Close main.py and return to the directory that contains main.py. You can then
 
 E. You may see a window open in your browser asking you to authorize the application. Do so - you will only have to do this once.
 
-F. Return to your terminal - you should see all of Taylor's albums printed out there.
\ No newline at end of file
+F. Return to your terminal - you should see all of Taylor's albums printed out there.
+
+## Accessing specific data
+The data returned is an object of type dictionary. Accessing the data is quite easy and then can be transformed into json and string format. Below is an example of what is returned from querying the API for a user's top tracks.
```

Comment:
> can be transformed into json and string format.

The wording could be improved. It reads like the user has to transform the json output themselves.

```suggestion
The data returned is an object of type dictionary. Accessing the data is quite easy as the library already parses the JSON response. Below is an example of what is returned from querying the API for a user's top tracks.
```